release-notes/opensearch.release-notes-1.3.9.md

```diff
@@ -0,0 +1,10 @@
+## 2022-03-15 Version 1.3.9 Release Notes
+
+### Upgrades
+- Upgrade `Netty` from 4.1.86.Final to 4.1.87.Final ([#6130](https://github.com/opensearch-project/OpenSearch/pull/6130))
```

Comment:
From logs under Dependencies, Netty seems to be bumped to 4.1.90.Final. Do we need to include that log here ?